using-valtimo/plugin/documenten-api/configure-documenten-api-plugin.md

```diff
@@ -58,6 +58,24 @@ An example process link configuration:
 
 ### Save uploaded document
 
-The **Save uploaded document.** saves a document to the Documenten API. Metadata provided by the user
+The **Save uploaded document** action, saves a document to the Documenten API. Metadata provided by the user
 through a form after uploading a file, is transferred together with the file to the Documenten API. After this plugin
-action has been selected, the user does not have to input any configuration data.
+action has been selected, the user does not have to input any configuration data. 
```

Comment:
Maybe rephrase to something like:

No configuration data has to be provided in order to configure this plugin action.